File: constants/theme.ts
// Brand colors extracted from app logo
// Sunset gradient: coral → orange → golden yellow
// Water: light blue → deep blue

export const Colors = {
  // Sunset gradient colors (top of logo)
  sunsetStart: "#FF5F45", // Coral red-orange
  sunsetMid: "#FF8C42", // Warm orange
  sunsetEnd: "#FFB347", // Golden yellow
  sunsetGlow: "#FFECD2", // Light glow at horizon

  // Water/wave colors (bottom of logo)
  waterLight: "#6BB3D9", // Light blue
  waterMid: "#4A90D9", // Medium blue
  waterDeep: "#2E5A88", // Deep blue
  waterDark: "#1A3A5C", // Darkest blue

  // App colors derived from brand
  // background: "#0F2744", // Deep night blue (darker than water)
  // surface: "#1A3A5C", // Water dark blue
  // surfaceLight: "#2E5A88", // Water deep blue
  background: "#151515", // Deep dark gray
  surface: "#2A2A2A", // Dark gray
  surfaceLight: "#404040", // Medium gray

  primary: "#FF6B4A", // Main coral/orange
  primaryLight: "#FF8C42", // Lighter orange
  primaryMuted: "#FF6B4A60", // Transparent coral

  secondary: "#4A90D9", // Water blue
  secondaryMuted: "#4A90D980", // Transparent blue

  text: "#FFFFFF", // White (like timer marks)
  textMuted: "#A8C5D9", // Muted blue-white
  textDark: "#6B8BA3", // Darker muted text

  accent: "#FFB347", // Golden yellow
  success: "#6BB3D9", // Light water blue
  warning: "#FF8C42", // Orange
  overtime: "#FF5F45", // Coral for overtime indicator
};

// Gradient definitions for use with expo-linear-gradient
export const Gradients = {
  sunset: ["#FF5F45", "#FF8C42", "#FFB347", "#FFECD2"],
  sunsetSimple: ["#FF6B4A", "#FFB347"],
  water: ["#6BB3D9", "#4A90D9", "#2E5A88"],
  waterDeep: ["#4A90D9", "#2E5A88", "#1A3A5C"],
  background: ["#1A3A5C", "#0F2744"],
  timerActive: ["#FF5F45", "#FF8C42"],
  timerOvertime: ["#FF5F45", "#FF3D2E"],
};

export const Spacing = {
  xs: 4,
  sm: 8,
  md: 16,
  lg: 24,
  xl: 32,
  xxl: 48,
};

export const FontSizes = {
  xs: 12,
  sm: 14,
  md: 16,
  lg: 20,
  xl: 28,
  xxl: 48,
  timer: 72,
};

export const BorderRadius = {
  sm: 8,
  md: 12,
  lg: 16,
  xl: 24,
  full: 9999,
};

File: watchOS/Chill Timer Watch App/Theme/Theme.swift
import SwiftUI

// Brand colors extracted from app logo
// Sunset gradient: coral -> orange -> golden yellow
// Water: light blue -> deep blue

struct AppColors {
    // Sunset gradient colors
    static let sunsetStart = Color(hex: "#FF5F45")    // Coral red-orange
    static let sunsetMid = Color(hex: "#FF8C42")      // Warm orange
    static let sunsetEnd = Color(hex: "#FFB347")      // Golden yellow
    static let sunsetGlow = Color(hex: "#FFECD2")     // Light glow at horizon

    // Water/wave colors
    static let waterLight = Color(hex: "#6BB3D9")     // Light blue
    static let waterMid = Color(hex: "#4A90D9")       // Medium blue
    static let waterDeep = Color(hex: "#2E5A88")      // Deep blue
    static let waterDark = Color(hex: "#1A3A5C")      // Darkest blue

    // App colors
    static let background = Color(hex: "#151515")     // Deep dark gray
    static let surface = Color(hex: "#2A2A2A")        // Dark gray
    static let surfaceLight = Color(hex: "#404040")   // Medium gray

    static let primary = Color(hex: "#FF6B4A")        // Main coral/orange
    static let primaryLight = Color(hex: "#FF8C42")   // Lighter orange
    static let primaryMuted = Color(hex: "#FF6B4A").opacity(0.4)

    static let secondary = Color(hex: "#4A90D9")      // Water blue

    static let text = Color.white
    static let textMuted = Color(hex: "#A8C5D9")      // Muted blue-white
    static let textDark = Color(hex: "#6B8BA3")       // Darker muted text

    static let accent = Color(hex: "#FFB347")         // Golden yellow
    static let overtime = Color(hex: "#FF5F45")       // Coral for overtime
}

struct AppGradients {
    static let sunset = LinearGradient(
        colors: [AppColors.sunsetStart, AppColors.sunsetMid, AppColors.sunsetEnd],
        startPoint: .top,
        endPoint: .bottom
    )

    static let sunsetSimple = LinearGradient(
        colors: [AppColors.primary, AppColors.sunsetEnd],
        startPoint: .top,
        endPoint: .bottom
    )

    static let water = LinearGradient(
        colors: [AppColors.waterLight, AppColors.waterMid, AppColors.waterDeep],
        startPoint: .topLeading,
        endPoint: .bottomTrailing
    )

    static let waterDeep = LinearGradient(
        colors: [AppColors.waterMid, AppColors.waterDeep, AppColors.waterDark],
        startPoint: .topLeading,
        endPoint: .bottomTrailing
    )

    static let timerActive = LinearGradient(
        colors: [AppColors.sunsetStart, AppColors.sunsetMid],
        startPoint: .top,
        endPoint: .bottom
    )

    static let timerOvertime = LinearGradient(
        colors: [AppColors.sunsetStart, Color(hex: "#FF3D2E")],
        startPoint: .top,
        endPoint: .bottom
    )
}

// Color extension for hex support
extension Color {
    init(hex: String) {
        let hex = hex.trimmingCharacters(in: CharacterSet.alphanumerics.inverted)
        var int: UInt64 = 0
        Scanner(string: hex).scanHexInt64(&int)
        let a, r, g, b: UInt64
        switch hex.count {
        case 3: // RGB (12-bit)
            (a, r, g, b) = (255, (int >> 8) * 17, (int >> 4 & 0xF) * 17, (int & 0xF) * 17)
        case 6: // RGB (24-bit)
            (a, r, g, b) = (255, int >> 16, int >> 8 & 0xFF, int & 0xFF)
        case 8: // ARGB (32-bit)
            (a, r, g, b) = (int >> 24, int >> 16 & 0xFF, int >> 8 & 0xFF, int & 0xFF)
        default:
            (a, r, g, b) = (255, 0, 0, 0)
        }
        self.init(
            .sRGB,
            red: Double(r) / 255,
            green: Double(g) / 255,
            blue: Double(b) / 255,
            opacity: Double(a) / 255
        )
    }
}
